sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -1732,9 +1732,9 @@ case class Left(str: Expression, len: Expression, child: Expression) extends Run
     Examples:
       > SELECT _FUNC_('Spark SQL ');
        10
-      > SELECT CHAR_LENGTH('Spark SQL ');
-       10
-      > SELECT CHARACTER_LENGTH('Spark SQL ');
+      > SELECT _FUNC_('Spark SQL');
+       9
+      > SELECT _FUNC_(binary('Spark SQL\\000'));
```

Comment:
The last two examples seems to show to users we also support `CHAR_LENGTH` and `CHARACTER_LENGTH `:
https://issues.apache.org/jira/browse/SPARK-20749

Reply:
ok. I will revert this but https://github.com/apache/spark/pull/25924#discussion_r327982434 instead of hard coded names in examples.